documentation/Developers.md

```diff
@@ -38,19 +63,22 @@ Before testing you need to install virtcontainers. The following command will in
 # sudo -E PATH=$PATH make install
 ```
 
+You also need to install some extra components required for testing, by running:
+```
+$ sudo -E PATH=$PATH utils/virtcontainers-setup.sh
+```
+
+> Note: this script installs components into **/tmp**. If you reboot your machine,
+> those files will *likely be removed*, and you will need to re-run the script again
+> before you can successfully run the tests again.
+
 To test `virtcontainers`, at the top level run:
 
 ```
-# make check
+# CI=true make check
```

Comment:
Could you change all the prompts to "`$ `" as "`# `" reads as "*OMG we're running as `root` dude!*" to me ;)